DMTN-107.tex

```diff
@@ -417,20 +428,20 @@ \subsection{Catalog Differencing}\label{ssec:potsol_catdiff}
+The options to produce alerts {\it without} proper templates and/or without using DIA, although they might at first seem like an easy solution to maximize thce number of alerts in LOY1, would actually require larger expansions of scope and engender more potential risks, and are hence disfavored.
+The  option to not produce alerts in LOY1 avoids all potential scope expansions, but is inconsistent with a high-level requirement set forth in the SRD and community expectations, and is not without its own risks, notably,  the missed opportunity to learn lessons about alert production early on, and potentially overloaded computational resources as the science community runs user-generated difference-imaging pipelines, and is hence equally disfavored. 
 These disfavored options furthermore fail to provide consistent data products to the users and, in some cases, also violate formal requirements on template images and/or alert contents.
 
-{\bf Recommendation:} Template images should be generated prior to the start of Operations from as much of the LSSTCam commissioning data as possible, and used for DIA and alert production during LOY1.
-To increase the area covered by commissioning data from $\sim$2000 $\rm deg^2$ (10\% of the WFD footprint), a "template-filler" observation scheduler should be prepared and used by the commissioning team whenever their activities are agnostic to imaging filter and telescope pointing, with the objective of covering as wide an area of sky as possible in at least one filter in order to maximize the number of {\em unique} alerts provided in LOY1.
-A moderate up-scope option to build templates during LOY1 as new data is acquired should also be attempted by DM on a best-effort basis, as this could increase the sky coverage by $\sim$1000-2000 $\rm deg^2$ per month.
+{\bf DM System Science Team Preferred Option for maximizing LSST science}: Template images should be generated prior to the start of Operations from as much of the LSSTCam commissioning data as possible, and used for DIA and alert production during LOY1.
+To increase the area covered by commissioning data from $\sim$2000 $\rm deg^2$ (10\% of the WFD footprint), if feasible given other constraints during commissioning, a "template-filler" observation scheduler should be prepared and used by the commissioning team whenever their activities are agnostic to imaging filter and telescope pointing, with the objective of covering as wide an area of sky as possible in at least one filter in order to maximize the number of {\em unique} alerts provided in LOY1.
+A moderate up-scope option to build rolling templates during LOY1 as new data is acquired should also be attempted by DM on a best-effort basis, as this could increase the sky coverage by $\sim$1000-2000 $\rm deg^2$ per month.
```

Comment:
I will brainstorm something other than "rolling"!

Reply:
Melissa and I did discuss it offline before the PR, I'm happy to take other suggestions. I want to convey the concept of dynamically building templates as new images become available. Similar a rolling/moving average - a new value is  calculated that replaces the old as and when new data becomes available

Reply:
My concerns are twofold--one is the semantic overlap with "rolling cadence", an unrelated concept.  The other issue is that it implies that individual templates at specific positions would be regenerated monthly, which may not be what's desired, as it makes the flux baselines jump around.  (Depending on the processing details, we may not be able to avoid it, though.)  

I would simply say "to build templates on a monthly basis during LOY1..."--no need for a special adjective.

Reply:
Or (see above) 
- A moderate up-scope option to build prefatory templates

Reply:
resolved by using "prefatory"